README.rst

```diff
@@ -0,0 +1,40 @@
+FogLAMP "random" C++ South plugin
+
+Build
+----
+To build FogLAMP Random C++ South plugin:
+- mkdir build
+- cd build
+- cmake ..
+
+- By default the FogLAMP develop package header files an libraries
+  are in use: /usr/include/foglamp and /usr/lib/foglamp
+- If FOGLAMP_ROOT env var is set and no -D options are set,
+  the header files and libraries paths are set to FOGLAMP_ROOT dir
+
+Options to pass to cmake:
+- FOGLAMP_SRC_DIR sets the path of a FogLAMP source tree
+- FOGLAMP_INCLUDES sets the path to FogLAMP header files
+- FOGLAMP_LIBS sets the path to FogLAMP libraries
+- FOGLAMP_INSTALL_DIR sets the installation path of Random plugin
+
+NOTE:
+ - FOGLAMP_INCLUDES must be used with FogLAMP header files all available in a single path
```

Comment:
"FOGLAMP_INCLUDES must be used with FogLAMP header files all available in a single path" ->
"The FOGLAMP_INCLUDES option should point to a location where all the FogLAMP header files have be installed in a single directory."

You also need to same something similar about FOGLAMP_LIBS.

Slightly inconsistency naming here as well, some have _DIR on the end some don't. Some are singular and other are plural. I would be tempted to drop the _DIR part and make them all singular.